frontend/ui/trapper.lua

```diff
@@ -181,6 +186,16 @@ function Trapper:info(text)
     -- events won't be considered action on the yet to be displayed
     -- widget
 
+  -- If fast_refresh option, avoid UIManager refresh overhead
```

Comment:
Total nitpick but wrong indentation is also immediately visible. :-P

Reply:
Yup, didn't strike me, I often do that in my own personal patch, to reduce the amount of diff :)

Reply:
I prefer tabs and to display tabs as two spaces but that's neither here nor there. ;-)